bundles/customer-registration/src/FondOfOryx/Zed/CustomerRegistration/CustomerRegistrationDependencyProvider.php

```diff
@@ -140,7 +141,6 @@ public function provideCommunicationLayerDependencies(Container $container): Con
         $container = $this->addLocaleFacade($container);
         $container = $this->addStoreFacade($container);
         $container = $this->addMailFacade($container);
-        $container = $this->addOneTimePasswordFacade($container);
```

Comment:
Don't we have another problem if we remove it in communication layer?

Reply:
i don think so, there ist no factory method for one-time-password-facade in the communication factory.